cmds/core/brctl/brctl.go

```diff
@@ -0,0 +1,192 @@
+// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// brctl - ethernet bridge administration brctl(8)
```

Comment:
Would it make sense to remove detailed commands from comments ? given "const usage = " already covers it.

plus, it is easy for people to neglect updating this comment

Reply:
I think that is a good idea. It could probably create more confusion than benefits.